third_party/flutter/bin/internal/shared.sh

```diff
@@ -23,18 +23,18 @@ function update_flutter {
   FLUTTER_STORAGE_BASE_URL=https://download.shorebird.dev $FLUTTER_PATH/bin/flutter --version  
 }
 
-function pub_upgrade_with_retry {
+function pub_get_with_retry {
```

Comment:
I took my time and read the whole bash script and realized that I shouldn't rename everything from upgrade to something else.

Since the function will still do more stuff and the whole goal of the function is indeed to upgrade the cli if needed, the `pub upgrade` was just part of the whole process.

So I ended up just replacing the the `upgrade` command to `get` and replace just the references that made sense.